SocialStudies/OrphanWells/orphan-wells.ipynb

```diff
@@ -112,7 +112,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
+    "url = 'http://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
```

Comment:
Please check the url. I get errors running the code cell both on Callysto hub and jupyter notebook.

Reply:
The URL works, but there's something about the SSL signing that is wrong. I didn't have any luck with the http (no s), but you can get it to work with the requests library first which allows you to ask it to ignore the SSL error. 

Working on a fix, can add the change this afternoon.

Reply:
@meghbali Can you verify that altering the cell as follows corrects the issue? 

```
url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'

#import ssl
#ssl._create_default_https_context = ssl._create_unverified_context

# import pandas as pd
# ww = pd.read_html(url)[1]
# ww.columns = ['UWI','Operator','County/Municipality','Meridian','Township','Range','Section','License Date','License Status']
# ww

import requests, warnings
import pandas as pd

with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    res = requests.get(url, verify=False)
ww = pd.read_html(res.text)[1]
ww.columns = ['UWI','Operator','County/Municipality','Meridian','Township','Range','Section','License Date','License Status']
ww
```

Reply:
@ZachAttacksProblems: Thanks. I implemented the changes you suggested and the issue is now resolved. I pushed a new commit.